shared/js/tab.js

```diff
@@ -147,21 +145,6 @@ class Tab {
     }
 }
 
-chrome.webRequest.onHeadersReceived.addListener((header) => {
-    let tab = tabManager.get({'tabId': header.tabId})
-
-    // Remove successful & rewritten requests
-    if (tab && header.statusCode < 400) {
-        tab.httpsRequests = tab.httpsRequests.filter((url) => {
-            // disregard diff between 'https://www.foo.com' and 'https://foo.com'
-            const _url = url.replace('https://www.', 'https://')
-            const _headerUrl = header.url.replace('https://www.', 'https://')
-            return _url !== _headerUrl
-        });
-    }
-
-}, {urls: ['<all_urls>']});
```

Comment:
Removed this, it was unused code. Probably extra overhead for every single request.